src/Utils/FormLib.js

```diff
@@ -0,0 +1,60 @@
+import { Visibility, VisibilityOff } from "@material-ui/icons";
+import { useField } from "formik";
+import { useState } from "react";
+import './FormLib.css'
+
+
+export const TextInput = ({icon, ...props}) =>{
+    const [field, meta] = useField(props)
+    const [show, setShow ] = useState(false)
+
+    return (
+        <div style={{position: "relative"}}>
+            <div className="contactes">
+                <label htmlFor={props.name}>
+                    {props.label}
+                </label>
+                <div className="input_icon_container">
+                    <span className="input_icon left">
+                        {icon}
+                    </span>
+                    {props.type !== "password" && 
+                    <input
+                        className="inputs" 
```

Comment:
To be honest am not feeling this, having two input method. look for a way you can use one

Reply:
I have resolved everything to one input now sir